documents/Code of Ethics.md

```diff
@@ -1,87 +1,97 @@
+      1. Competition budgets must be held by the Delegate for a period of 7 years and can be requested for review by the WCA Board, WEC, WFC or the Senior Delegate.
       2. The budget of a competition should be in alignment with the Mission and Spirit of the WCA.
-      3. Profit acquired from a competition must not be kept for personal gain and should be put towards the benefit of the community.
-      4. Listed Delegates must advise organizers to take into account WCA Dues when making the budget and comply with Finance invoices.
-2. **Communication of WCA Staff**
-   1. WCA Staff are expected to respond to email within 48 hours.
-   2. WCA Staff must only use @worldcubeassociation.org email addresses for WCA-related functions. These email accounts must not be used for external logins that are not related to WCA work.
+      3. Profit acquired from a competition must not be kept for personal gain and should be put towards the benefit of the community. Where possible this should be done via an organization, and not an individual.
+         1. If profits from a competition will be donated to a charity, this information must be made public on the competition page when it is announced. Any such charity must be registered as such in the country the competition is held.
+      4. Listed Delegates comply with Finance invoices and must advise Organizers to take into account WCA Dues when making the budget.
+2. **Communication**
+   1. The [WCA email account policy](https://www.worldcubeassociation.org/documents/policies/internal/Email%20Account.pdf) must be followed.
       1. All official WCA email communication must utilize the @worldcubeassociation.org addresses.
       2. WCA Staff must be aware that emails sent from @worldcubeassociation.org email addresses to external parties may be perceived as being on behalf of the WCA. Personal email addresses should be used to avoid this perception.
          1. Example: Contacting sponsors for a competition should use personal email addresses.
-   3. WCA Delegates must communicate with other Delegates in their region and/or in any region they are planning to Delegate a competition.
-   4. All work done within the WCA is confidential and must not be disclosed to non-WCA Staff members without explicit permission from relevant members.
+   2. WCA Delegates must communicate with other Delegates in their region and/or in any region they are planning to delegate a competition.
+   3. All work done within the WCA is confidential and must not be disclosed to non-WCA Staff members without explicit permission from relevant members.
+      1. Delegates may provide relevant feedback to the Organizers but must not give visibility of incidents, or sections written by other Delegates.

```

Comment:
I'm blanking on a re-word for that section but I don't infer the same as you, this allows copy/pasting of sections such as "Organization" instead of the Delegate having to re-write something if they want to give feedback to the Organizer, reducing their workload if this is requested.

I don't understand why we need to make an exception for other Staff members seeing reports when the phrasing is "must not be disclosed to non-WCA Staff members".

As for training Delegates, this definitely seems like something WQAC would be able to produce. A kind of dummy report with mock incidents etc. For incident context and reasoning, surely the best time to do that would be at the competition when the incident has just occured.

Reply:
I agree that having a dummy report produced by WQAC would be useful. However I see no reason why a typical incidents section of a delegate report cannot be shared to organisers (with the removal of specific names, or sensitive incidents, as required). It is a clear way of showing what incidents are/aren't worth highlighting, in which circumstances to seek WRC feedback, and how to clearly articulate an incident and the outcome.

I had missed that this was a subpoint to something that clearly stated "non-WCA Staff". I agree no exception then is required then.

Reply:
1. Delegates may provide relevant feedback to the Organizers but must not give visibility of **sensitive information**, or sections written by other Delegates.

Thoughts on that phrasing?

Reply:
I much prefer this phrasing. I think it gives the flexibility to delegates to help organisers, while keeping the correct information around privacy etc.

Reply:
```suggestion
      1. Delegates may provide relevant feedback to the Organizers but must not give visibility of sensitive information, or sections written by other Delegates.
```